examples/plot_clickable_image.py

```diff
@@ -0,0 +1,70 @@
+"""
+=========================================
+Demonstration of how to use ClickableImage / generate_2d_layout.
+=========================================
+
+In this example, we open an image file, then use ClickableImage to
+return 2D locations of mouse clicks (or load a file already created).
+Then, we use generate_2d_layout to turn those xy positions into a layout
+for use with plotting topo maps. In this way, you can take arbitrary xy
+positions and turn them into a plottable layout.
+"""
+# Authors: Christopher Holdgraf <choldgraf@berkeley.edu>
+#
+# License: BSD (3-clause)
+import sys
+sys.path.insert(0, '../')
```

Comment:
please remove this. BTW if you need this because you need to import mne-python, do `python setup.py develop`. It basically symlinks your dev version of mne-python so when you do `import mne` it imports whatever you currently have checked out via `git`.